app/models/post.rb

```diff
@@ -0,0 +1,6 @@
+class Post < ActiveRecord::Base
+  belongs_to :user
+  acts_as_taggable
+  has_many :comments
+  has_many :follows
```

Comment:
you should probably add `dependent: :destroy` to these relationships so that when a `Post` is deleted, these are as well - otherwise you end with dangling references in your DB